pkg/test/utils/kubernetes.go

```diff
@@ -392,6 +392,9 @@ func ConvertUnstructured(in runtime.Object) (runtime.Object, error) {
 	// Deprecated: kudo.dev is deprecated.
 	kudoGroup := "kudo.dev"
 	kuttlGroup := "kuttl.dev"
+	if group == kudoGroup {
```

Comment:
with this change, this warning is printed for all the resources from KUDO (try running KUDO tests now and you'll see). I am not sure this was the right place to put this warning to.

Reply:
👍 
This function is also called when applying general resources. These could be KUDO CRDs or CRs where `kudo.dev` is expected.

Reply:
yeah this is what kudo tests say right now
```
2020/10/26 13:56:54 WARNING: kudo.dev group has been deprecated and is schedule to be removed in KUTTL 0.8.0
2020/10/26 13:56:54 WARNING: kudo.dev group has been deprecated and is schedule to be removed in KUTTL 0.8.0
```